src/logic/json-parsing/get-member-type.ts

```diff
@@ -28,7 +28,15 @@ export const getMemberType = (
     }
 
     if (property.enum) {
-      return property.enum.map((el) => `'${el}'`).join(' | ');
+      return property.enum
+        .map((el) => {
+          if (isNaN(parseInt(el.toString(), 10))) {
```

Comment:
Maybe you should check for `property.type === 'number'`?

Reply:
<img width="604" alt="image" src="https://github.com/hublo/swagger-typescript-types/assets/78426796/1a9cd1dc-65f7-452c-8c67-2caf6ba7bdaa">

Reply:
i will check "string" or "number" or unknown values (other than "string" and "number")

Reply:
see together, keep my code like this in order to manage also heterogeneous enums (contains number and string values)